src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -198,6 +268,63 @@ export class MoreInfoDialog extends LitElement {
     );
   }
 
+  private async _getLogBookData() {
+    if (!this._entityId) {
+      return;
+    }
+    this._isLoading = true;
+
+    const yesterday = new Date(new Date().getTime() - 24 * 60 * 60 * 1000);
+    const now = new Date();
+    const [entries] = await Promise.all([
+      getLogbookData(
```

Comment:
We will need to teach `getLogbookData` to pass the `entity_matches_only ` https://github.com/home-assistant/core/pull/39555

Reply:
Sounds good. Thanks @bdraco